sklearn/ensemble/_bagging.py

```diff
@@ -87,7 +87,7 @@ def _parallel_build_estimators(n_estimators, ensemble, X, y, sample_weight,
                                              random_state=random_state)
 
         # Draw random feature, sample indices
-        features, indices = _generate_bagging_indices(random_state,
+        features, indices = _generate_bagging_indices(seeds[i],
```

Comment:
I would also like that `ensemble._make_estimator` to take a seed instead of a random state. Like this, we will internally create the `random_state` instead of passing it around.

Reply:
Good idea. Replacing the 85th line `random_state = np.random.RandomState(seeds[i])` into `random_state = seeds[i]` would be a solution.